### PR TITLE
Update Docs: Configuration “beta-features”

### PR DESCRIPTION
**Pull request by martinjagodic**
Automatically generated by Netlify CMS

### CODE REVIEW

File: website/content/docs/beta-features.md [hunk 1]
```diff
@@ -614,3 +612,5 @@ collections:
     # moving an existing entry will move the entire sub tree of the entry to the new location
     meta: { path: { widget: string, label: 'Path', index_file: 'index' } }
 ```
+
+**Note:** Nested collections work poperly only with `slug/index.md` directory structure. If you use `slug.md` approach, you might see some unwanted results.
```
Comment (erezrokah) [inline website/content/docs/beta-features.md:616]: I'm not sure what `slug.md` approach means.
Could we rephrase it as `Nested collections expect the following directory structure....` and provide something like this:
```
content
└── pages
    ├── authors
    │   ├── author-1
    │   │   └── index.md
    │   └── index.md
    ├── index.md
    └── posts
        ├── hello-world
        │   └── index.md
        └── index.md
```
Will that make it more clear?

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @martinjagodic for the PR. Good point about clarifying the expected structure.
Please let me know what you think about my comment.

File: website/content/docs/beta-features.md (hunk 1 above)
Comment (martinjagodic) [reply website/content/docs/beta-features.md]: @erezrokah thanks for your review. Your suggestion makes more sense, so I changed the content accordingly. I spent quite some time dealing with this before I realized that this structure is required. So I opened this PR to maybe save someone else's time.

Would it be possible to make this work with this sort of structure? It's what I meant with `slug.md`. If it's possible, I would be glad to open an issue, maybe even a pull request.

```
content
└── pages
    ├── authors
    │   ├── author-1.md
    │   └── index.md
```

Comment (erezrokah) [reply website/content/docs/beta-features.md]: > Would it be possible to make this work with this sort of structure? It's what I meant with `slug.md`. If it's possible, I would be glad to open an issue, maybe even a pull request.

Opening an issue will be a great start. A pull request will be 🔥

**Review by erezrokah — APPROVED** [review]

**Pull request merged**